mongo.go

```diff
@@ -123,7 +134,15 @@ func (m Handler) Update(ctx context.Context, item *resource.Item, original *reso
 		return err
 	}
 	defer m.close(c)
-	err = c.Update(bson.M{"_id": original.ID, "_etag": original.ETag}, mItem)
+	s := bson.M{"_id": original.ID}
+	if strings.HasPrefix(original.ETag, "p-") {
+		// If the original ETag is weak,
```

Comment:
We are no longer talking about weak etags here. Please fix comment.

Reply:
Will fix.